5 - RPG/Code/ClassLibrary1/Abilities/AbilityPoints.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Characters
+{
+    public abstract class AbilityPoints
+    {
+
+            public string AbilityName { get; set; }
+            public int Min { get; set; }
+            public int Max { get; set; }
+            public int Value { get; set; }
+
+            public string Validate()
```

Comment:
This method name is a bit misleading. Typically validation methods either return a `bool` to state that the object is valid or not. This method here works more like a `GetValidationFailureMessage` 
I would consider separating these into 2 methods: One to check `IsValid` and the second to get the failure message if needed. This will help avoid sending the empty string